extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -197,10 +197,12 @@ require([
 
 	$(function () {
 		$('.embeddable-discussions-module').on('click', '.upvote', function (event) {
-			var upvoteUrl = getBaseUrl() + event.currentTarget.getAttribute('href'),
+			var postId = event.currentTarget.getAttribute('data-id'),
 				hasUpvoted = event.currentTarget.getAttribute('data-hasUpvoted') === '1',
-				$svg = $($(event.currentTarget).children()[0]),
-				verb = hasUpvoted ? 'DELETE' : 'POST';
+				method = hasUpvoted ? 'downVotePost' : 'upVotePost',
+				upvoteUrl = mw.config.get('wgScriptPath') + 
```

Comment:
I suggest using https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams to build query params.